modules/cloudaccess/main.tf

```diff
@@ -1,10 +1,4 @@
 locals {
-  resource_suffix = var.resource_suffix
-
-  # an AWS IAM OIDC "provider" is NOT required when the trust is to google, nor when the user
-  # tells us they're managing it themselves, only when provisioning for an Azure client + tenant ID
-  create_azure_trust  = var.wayfinder_identity_azure_client_id != "" && var.wayfinder_identity_azure_tenant_id != "" ? true : false
-  create_google_trust = var.wayfinder_identity_gcp_service_account != "" ? true : false
-
-  azure_oidc_issuer = local.create_azure_trust ? "https://sts.windows.net/${var.wayfinder_identity_azure_tenant_id}/" : ""
+  resource_suffix   = var.resource_suffix != "" ? "-${var.resource_suffix}" : ""
+  azure_oidc_issuer = "https://sts.windows.net/${var.wayfinder_identity_azure_tenant_id}/"
```

Comment:
will this work if wayfinder_identity_azure_tenant_id is unspecified? if not, this needs to be guarded with var.from_azure == true...

Reply:
I'll add your check back in, as it's better to fail with creating an oidc issuer with an empty string (more obvious error).